contentcuration/contentcuration/view/internal_views.py

```diff
@@ -238,21 +238,27 @@ def convert_data_to_nodes(content_data, parent_node):
     try:
         root_mapping = {}
         sort_order = 1
+        parent = ContentNode.objects.get(pk=parent_node)
+        existing_node_ids = parent.children.values_list('node_id', flat=True) if parent else []
```

Comment:
I think you can replace these two lines with just `existing_node_ids = ContentNode.objects.filter(parent_id=parent_node).values_list('node_id', flat=True)`.